pkg/kubectl/cmd/create_service.go

```diff
@@ -83,6 +82,10 @@ func NewCmdCreateServiceClusterIP(f cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	return cmd
 }
 
+func errUnsupportedGenerator(cmd *cobra.Command, generatorName string) error {
```

Comment:
I think it's not necessary write a new function which only use in this package, or like @mengqiy suggest move this helper function out to util